app/app.js

```diff
@@ -34,7 +34,7 @@ openSansObserver.check().then(() => {
 });
 
 // Import the CSS reset, which HtmlWebpackPlugin transfers to the build folder
-import '../node_modules/sanitize.css/dist/sanitize.min.css';
+import '../node_modules/sanitize.css/sanitize.css';
```

Comment:
Download it and unzip it in your root component. :grin:

@okonet wrote that, have to ask him!